test/conformance/kuttl/verifyImages/clusterpolicy/standard/noconfigmap-diffimage-success/policy.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  annotations:
+    pod-policies.kyverno.io/autogen-controllers: none
+  name: image-verify-polset
+spec:
+  background: false
+  failurePolicy: Fail
+  rules:
+  - context:
+    - configMap:
+        name: myconfigmap
+        namespace: mynamespace
```

Comment:
Where do we create this configmap? 

I only see `mynamespace/myconfigmap1` in test/conformance/kuttl/verifyImages/clusterpolicy/standard/noconfigmap-diffimage-success/pod-with-configmap.yaml.

Reply:
typo here. will fix it

Reply:
Hmm I was expecting these two to be added:
>2. the same test in the proof manifest, with the configmap created. And the image should be verified by kyverno.
>3. Kyverno policy is not blocked when the configmap is not present in the cluster

Can you summarize the tests you added?

Reply:
steps followed in tests:
- create a policy with `ghcr.io/*` image in imageRef (don't create configmap)
- create a resource with `nginx:latest` image (this should be succeeded)
- create a resource with `ghcr.io/kyverno/test-verify-image:signed` image (this should failed as configmap is not created)
- update policy by changing configmap name from `myconfigmap` to `myconfigmap1`
- create configmap `myconfigmap1`
- create a resource with `ghcr.io/kyverno/test-verify-image:signed` image (this should now succeed)

Reply:
This looks good!